jbpm-wb-dashboard/jbpm-wb-dashboard-client/src/main/java/org/jbpm/dashboard/renderer/client/panel/ProcessDashboard.java

```diff
@@ -337,19 +339,27 @@ public void tableCellSelected(String columnId,
         Integer processInstanceStatus = Double.valueOf(ds.getValueAt(rowIndex,
                                                                      COLUMN_PROCESS_STATUS).toString()).intValue();
 
-        openProcessDetailsScreen();
-        setupDetailBreadcrumb(i18n.processDashboardName(),
-                              i18n.ProcessInstanceBreadcrumb(processInstanceId),
-                              PROCESS_INSTANCE_DETAILS_SCREEN);
-
-        instanceSelectionEvent.fire(new ProcessInstanceSelectionEvent(
-                deploymentId,
-                processInstanceId,
-                processDefId,
-                processDefName,
-                processInstanceStatus,
-                true,
-                serverTemplateSelectorMenuBuilder.getSelectedServerTemplate()));
+        specManagementService.call((ServerTemplate serverTemplate) -> {
+            if (serverTemplate.getContainerSpec(deploymentId).getStatus().equals(KieContainerStatus.STARTED)) {
```

Comment:
Could it be that the containerspec is null in case the container was removed/deleted? In that chase would be best to check before trying to access the container status.